test/y2network/hostname_test.rb

```diff
@@ -98,40 +98,4 @@
       end
     end
   end
-
-  describe "#save_hostname?" do
-    context "When used in installer" do
-      before(:each) do
-        allow(Yast::Stage).to receive(:initial).and_return(true)
-      end
-
-      context "without explicitly set hostname" do
```

Comment:
is this logic somehow kept working or not needed anymore?

Reply:
What do you mean exactly?:

Now the static hostname field honors the /etc/hostname content, so, modify it in case it is modified through the dialog.

1. **In case that the hostname is given by linuxrc**

    it is written by the InstallInfCovertor at https://github.com/yast/skelcd-control-leanos/blob/master/control/control.leanos.xml#L514

2. **If it is not given**

    No hostname is proposed at all. In case of dhcp setup, the hostname is modified by wicked, but we will honor the content of the static hostname. So, no hostname will be proposed and at the end of the installation.


Thus, basically what you described at: https://github.com/yast/yast-network/blob/master/doc/hostname.md#installation-proposals

Reply:
Ah, i put it to bad line. I meant logic when hostname is set up via linuxrc option. https://github.com/yast/yast-network/pull/1114/files#diff-699f24097595d5e63d94b1c8bbc3ff96L78